viral/cache_2p_sessions.py

```diff
@@ -127,11 +134,13 @@ def add_daq_times_to_trial(
     for state in trial.states_info:
         state.start_time_daq = bpod_to_daq(state.start_time).astype(float)
         state.end_time_daq = bpod_to_daq(state.end_time).astype(float)
-        state.closest_frame_start = int(
-            np.argmin(np.abs(valid_frame_times - state.start_time_daq))
+        state.closest_frame_start = (
+            int(np.argmin(np.abs(valid_frame_times - state.start_time_daq)))
+            + offset_after_pre_epoch
         )
-        state.closest_frame_end = int(
-            np.argmin(np.abs(valid_frame_times - state.end_time_daq))
+        state.closest_frame_end = (
+            int(np.argmin(np.abs(valid_frame_times - state.end_time_daq)))
+            + offset_after_pre_epoch
```

Comment:
Can you add a check to test that neither the `closest_frame_start` or `closest_frame_end` are within the pre or post freeze periods?